cass-config-builder.yaml

```diff
@@ -0,0 +1,52 @@
+package:
+  name: cass-config-builder
+  version: 1.0.7
+  epoch: 0
+  description: |
+    Configuration builder for Apache Cassandra based on definitions at datastax/cass-config-definitions
+  copyright:
+    - license: Apache-2.0
+  dependencies:
+    runtime:
+      - openjdk-8
+
+environment:
+  contents:
+    packages:
+      - build-base
+      - busybox
+      - ca-certificates-bundle
+      - openjdk-8-default-jvm
+
+pipeline:
+  - uses: git-checkout
+    with:
+      repository: https://github.com/datastax/cass-config-builder
+      expected-commit: 537bbf240a0ed3754ea0f73db23e411169fe0676
+      tag: v${{package.version}}
+
+  - uses: patch
+    with:
+      # to mitigate the CVEs related to the package
+      patches: deps.patch
+
+  - runs: |
+      sed -i 's|git@\(.*\):|https://\1/|g' .gitmodules
+      cat .gitmodules
+      git submodule update --init
+
+  - runs: |
+      export JAVA_HOME=/usr/lib/jvm/java-1.8-openjdk
+      ./gradlew copyDockerBuildCtx
```

Comment:
I'm not sure if this is actually compiling the sources?  

```
ℹ️  aarch64   | Downloading https://services.gradle.org/distributions/gradle-6.0.1-bin.zip
ℹ️  aarch64   | ..........................................................................................
ℹ️  aarch64   | 
ℹ️  aarch64   | Welcome to Gradle 6.0.1!
ℹ️  aarch64   | 
ℹ️  aarch64   | Here are the highlights of this release:
ℹ️  aarch64   |  - Substantial improvements in dependency management, including
ℹ️  aarch64   |    - Publishing Gradle Module Metadata in addition to pom.xml
ℹ️  aarch64   |    - Advanced control of transitive versions
ℹ️  aarch64   |    - Support for optional features and dependencies
ℹ️  aarch64   |    - Rules to tweak published metadata
ℹ️  aarch64   |  - Support for Java 13
ℹ️  aarch64   |  - Faster incremental Java and Groovy compilation
ℹ️  aarch64   |  - New Zinc compiler for Scala
ℹ️  aarch64   |  - VS2019 support
ℹ️  aarch64   |  - Support for Gradle Enterprise plugin 3.0
ℹ️  aarch64   | 
ℹ️  aarch64   | For more details see https://docs.gradle.org/6.0.1/release-notes.html
ℹ️  aarch64   | 
ℹ️  aarch64   | Starting a Gradle Daemon (subsequent builds will be faster)
ℹ️  aarch64   | > Task :compileJava NO-SOURCE
ℹ️  aarch64   | > Task :processResources NO-SOURCE
ℹ️  aarch64   | > Task :checkClojure
ℹ️  aarch64   | > Task :compileClojure
ℹ️  aarch64   | > Task :classes
ℹ️  aarch64   | > Task :shadowJar
ℹ️  aarch64   | > Task :copyDockerBuildCtx
ℹ️  aarch64   | 
ℹ️  aarch64   | BUILD SUCCESSFUL in 25s
ℹ️  aarch64   | 4 actionable tasks: 4 executed
ℹ️  aarch64   | retrieving workspace from builder: 
ℹ️  aarch64   | retrieved and wrote post-build workspace to: /tmp/melange-workspace-1338470849
ℹ️  aarch64   | running package linters for cass-config-builder
⚠️  aarch64   | WARNING: /usr/local path found in non-compat package
⚠️  aarch64   | WARNING: /usr/local path found in non-compat package
⚠️  aarch64   | WARNING: /usr/local path found in non-compat package
⚠️  aarch64   | WARNING: /usr/local path found in non-compat package
⚠️  aarch64   | WARNING: /usr/local path found in non-compat package
```

I'm not familiar with cass-config-builder but wonder if `./gradlew assemble` is what we want to use and move files from ./build/libs/ into the ${{targets.contextdir}}?

Reply:
I followed the instructions within the [Dockerfile](https://github.com/datastax/cass-config-builder/blob/master/docker/Dockerfile) and I looked at the GitHub Actions workflow for releasing the artifacts and found this [line](https://github.com/datastax/cass-config-builder/blob/537bbf240a0ed3754ea0f73db23e411169fe0676/.github/workflows/release.yml#L32) and this line tells me that its using the script [here](https://github.com/datastax/cass-config-builder/blob/537bbf240a0ed3754ea0f73db23e411169fe0676/scripts/push-release.sh) and this script does only `docker build` nothing else, so I didn't use the `assemble` command because I couldn't see it anywhere else in the release process but yeah I'm not familiar with the project either, so, if someone helps us in here that'd be great maybe @mamccorm <3

Reply:
I just checked the two the only difference between to is:

- copyDockerBuildCtx creates `cass-config-builder-1.0.7-20231201-all.jar``
- assemble creates `cass-config-builder-1.0.7-20231201.jar`

but according to their Dockerfile, they also use `copyDockerBuildCtx`, so it'd be easier to follow their process, WDYT @rawlingsj?

Reply:
Lets go with alignment for now, but happy to change / tweak again if theres any more concerns